config/locales/en/placements/providers/find.yml

```
@@ -88,27 +88,26 @@ en:
               expected_date: Expected date
         details_navigation:
           page_title: "%{school_name} - Find"
+          email_this_school: Email this school if you have suitable trainees
```

```suggestion
          email_this_school: Email this school if you have suitable trainees.
```

Can we add the missing full stop here?